src/highgui.rs

```diff
@@ -114,3 +117,20 @@ pub enum MouseEventType {
     /// Positive/negative means right and left scrolling.
     MouseHWheel = 11,
 }
+
+/// Provides some highgui functionallity
+pub trait Shower {
```

Comment:
Is not the best name

Reply:
Verb may be better, like `Show`.
Or just `HighGui` or `Gui`?